net-libs/libwebsockets/libwebsockets-3.0.0.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="C library for lightweight websocket clients and servers"
+DESCRIPTION="Libwebsockets (LWS) is a flexible, lightweight pure C library \
+for implementing modern network protocols easily with a tiny footprint, \
+using a nonblocking event loop."
+HOMEPAGE="https://libwebsockets.org/"
+COPYRIGHT="2010-2018 Andy Green"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/warmcat/libwebsockets/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="a6b611c212c52f161f70556339fdaa199b7e9b6a167c4638e086d19db75d6290"
+SOURCE_DIR="libwebsockets-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	libwebsockets$secondaryArchSuffix = $portVersion
+	lib:libwebsockets$secondaryArchSuffix = $portVersion
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libwebsockets${secondaryArchSuffix}_devel = $portVersion
+	devel:libwebsockets$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libwebsockets$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcrypto$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:cmake
+	cmd:make
+	cmd:openssl
+	"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	
+	cmake \
+		-DCMAKE_INSTALL_PREFIX=$prefix \
+		-DCMAKE_BUILD_TYPE=release \
+		-DLWS_INSTALL_INCLUDE_DIR=$includeDir \
+		-DLWS_INSTALL_LIB_DIR=$libDir \
+		-DLWS_WITHOUT_EXTENSIONS=OFF \
+		-DLWS_WITHOUT_TESTAPPS=ON \
+		-DLWS_WITH_HTTP2=ON \
+		-DLWS_WITH_CGI=ON \
+		-DLWS_IPV6=ON \
+		-DLWS_WITH_ZIP_FOPS=ON \
+		-DLWS_WITH_RANGES=ON \
+		-DLWS_WITH_ACME=ON \
+		-DLWS_WITH_SERVER_STATUS=ON \
+		..
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd build
+	make install
+	
```

Comment:
```diff
 	make install
-	
+
-	prepareInstalledDevelLibs libwebsockets
+	prepareInstalledDevelLib libwebsockets

```
* Unneeded trailing tab.
* Since we know there is only one lib in this recipe and that it is unlikely that we will have more than one lib in the future, we can use prepareInstalledDevel**Lib** instead of prepareInstalledDevel**Libs**.

That said, prepareInstalledDevel**Libs** is also OK, and I have to admit that many recipes use prepareInstalledDevel**Libs** although they have only one lib, so you can keep this unchanged if you wish :)